### PR TITLE
Initial version of search bar with basic layout and functionality

### PR DESCRIPTION
**Pull request by ztolley**

### CODE REVIEW

File: stylesheets/_searchbar.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
+.searchbar {
+
+  .searchbar-content {
+    position: relative;
+    background: $white;
+  }
+
+  .searchbar-label {
+    float: none;
```
Comment (teneightfive) [inline stylesheets/_searchbar.scss]: does it need this?

File: stylesheets/application.scss [hunk 2]
```diff
@@ -18,6 +18,5 @@ $images-dir: '/images/';
 @import 'tabs';
 @import 'form';
 @import 'buttons';
-@import 'results-list';
-
+@import 'searchbar';
```
Comment (teneightfive) [inline stylesheets/application.scss]: Seems this conflict wasn't resolved properly. Don't think we want to remove `results-list`.

File: views/ukti.jade [hunk 3]
```diff
@@ -9,13 +9,11 @@ block global_subheader
...
+  li Built by
     a(href="https://digitaltrade.blog.gov.uk/")
       abbr(title="UK Trade & Investment")  UKTI
     = Digital
 
 block stylesheets
   link(rel='stylesheet', href='/stylesheets/application.css')
 
-block main_js
-  script(src='/javascripts/app.bundle.js')
```
Comment (teneightfive) [inline views/ukti.jade]: Is this because it's already being loaded as part of the template?

File: stylesheets/_searchbar.scss (hunk 1 above)
Comment (ztolley) [inline stylesheets/_searchbar.scss]: yes

File: views/ukti.jade (hunk 3 above)
Comment (ztolley) [inline views/ukti.jade]: yes

**Pull request merged**